### PR TITLE
Added clarifying notes for workshop chairs

### PR DESCRIPTION
**Pull request by mjpost**

### CODE REVIEW

File: hugo/content/info/contrib.md [hunk 1]
```diff
@@ -50,6 +48,7 @@ Please note the following important dates.
 ### Register your meeting
 
 If you are a conference publications chair, you must register your intention to submit your proceedings.
+(Workshop chairs should do this
```
Comment (akoehn) [inline hugo/content/info/contrib.md:51]: incomplete sentence

**Review by akoehn — COMMENTED** [review]
I suspect there is a reason for this change :-)

File: hugo/content/info/contrib.md (hunk 1 above)
Comment (mjpost) [reply hugo/content/info/contrib.md]: I seem unable to write anything without glaring typographical and other errors.

**Comment by mjpost** [comment]
> I suspect there is a reason for this change :-)

I'd call it a "clarification" instead of a "change" :)

**Review by akoehn — APPROVED** [review]
I approve of this clarification

**Pull request merged**